test/release/examples/primers/arrays.chpl

```diff
@@ -372,18 +370,26 @@ forall (i,j) in ProbSpace do
 
 writeln("Y is:\n", Y);
 
+/*
+   Our current implementation does not yet support arrays of arrays
```

Comment:
open question: Should we include unimplemented features in the primers? If not, it'd still be worth mentioning that Chapel currently does *not* support skyline arrays.